app/models/manageiq/providers/cisco_intersight/inventory/parser/physical_infra_manager.rb

```diff
@@ -23,10 +23,10 @@ def physical_servers
         # Since Intersight's source object is only consolidated view of either object ComputePhysicalSummary of ComputeRackUnit,
         # source object has to be obtained. I store it onto source_object
         source_object = collector.get_source_object_from_physical_server(server)
-        board_unit = collector.get_compute_board_by_moid(source_object.board.moid)
+        board_unit = collector.get_compute_board_by_moid(source_object.board.moid) if source_object.board
```

Comment:
Not sure if it is possible for `source_object` to be `nil` here, but `if source_object&.board` would handle that if it were

Reply:
I don't hink it can, but I can surely add this right away.

Reply:
Fixed in the updated commit.